.travis_scripts/build_all

```diff
@@ -29,7 +30,7 @@ echo ""
 echo "Configuring conda."
 conda config --add channels conda-forge
 conda config --set show_channel_urls true
-conda install --yes --quiet conda-forge-ci-setup=1.* conda-forge-pinning networkx conda-build=3.16
+conda install --yes --quiet conda-forge-ci-setup=2.* conda-forge-pinning networkx conda-build
```

Comment:
Relaxing the `conda-build` version should go in a separate PR. I think all issues that caused this pinning were resolved, but it would be good to check with core in that PR in case others know of issues.

Reply:
The last issue I remember was Go related. If you think we are good to unpin it, am happy to move forward with that PR.